app/managers.py

```diff
@@ -0,0 +1,35 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db")
+        self._table_name = "actors"
+
+    def all(self) -> list:
+        sql = (f"SELECT id, first_name, last_name "
```

Comment:
You can skip creating additional variable in every function. But if you thinks it looks better, it is ok